packages/delivery-node/delivery.js

```diff
@@ -23,6 +23,7 @@ module.exports = () => ({
       })
     } catch (e) {
       logger.error(e)
+      cb(e)
     }
   },
   sendSession: (logger, config, session, cb = () => {}) => {
```

Comment:
the value of `cb` could be `noop` by default, for additional consistency

Reply:
Yeah I see I've been inconsistent with `() => {}` and `noop`. Might be good to do this in a single pass once all of the branches have converged again?